Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM amazonlinux
+
+RUN yum install -y ruby23-devel git gcc gcc-c++ postgresql-devel zlib-devel
+
+RUN gem install bundler
+
+ENV INSTALL_PATH /test_track
+
+RUN mkdir -p $INSTALL_PATH
+
+WORKDIR $INSTALL_PATH
+
+COPY Gemfile Gemfile.lock ./
+
+COPY . .
+
+RUN bundle install --binstubs
+
+CMD puma -C config/puma.rb
```

Comment:
Mostly because I wasn't looking at that site when I wrote the Dockerfile :-). I think this should still get us to where we want to go though. After installing the deps (lines 1 - 5), the RUN and WORKDIR commands are nearly identical and just use an install path defined in line 7. I'm preferring the COPY instruction to the ADD instruction since we don't need to use ADD's URL expansion or decompression capabilities, and COPY is a bit more transparent as outlined here: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy. The ordering of copying the gem and lock files, running bundle install, and copying the app directory shouldn't matter I don't think. Lastly, the CMD instruction was just to provide an entrypoint in case we wanted to run this standalone using the Docker CLI instead of docker-compose (ie, if we wanted to do something like use a local instance of postgres instead of a containerized version).